### ISSUE TITLE
Add utils test

### ISSUE THREAD

**Issue by dai-shi** [pmndrs/jotai#87]
We don't have any tests for jotai/utils. Would someone work on it?

**Comment by LuckyPigeon** [comment]
I can do this, and can also try to work on #88, since they are both belongs to the test category.

**Comment by dai-shi** [comment]
@merisbahti 
https://github.com/pmndrs/jotai/pull/91#issue-497444938
> I suggest the convention utils.${function}.test.tsx or similar, how do you feel about this?

I prefer splitting utils.
```
./src/utils/index.ts
./src/utils/useUpdateAtom.ts
./test/utils/useUpdateAtom.test.tsx
```

How do you guys feel?

**Comment by LuckyPigeon** [comment]
Looks good to me

**Comment by dai-shi** [comment]
OK, I will create a PR to split src/utils.
@merisbahti started working on testing useUpdateAtom.
@LuckyPigeon would you like to start looking into #88?

**Comment by merisbahti** [comment]
Absolutely.

**Comment by LuckyPigeon** [comment]
Sure

**Comment by dai-shi** [comment]
Closed by #98 #99 #100 #101
